### DISCUSSION TITLE
Not receiving roles

### DISCUSSION THREAD

**Discussion by CringeThe** [danthe1st/Mee6-bypasser#60] [Support] [answered]
Bot has all perms, is at the top of the role hierarchy, all commands function as well. When it comes to actually adding the roles at the desired level, it fails to do so.

**Comment by danthe1st** [comment, marked as answer]
We just rolled an update that is supposed to fix this.
If it still does not work, check whether all text on your rank card is readable (the colour of the rank card text should not be similar to the background) and that there are no special characters.

**Reply by CringeThe** [reply]
Thank you! Everything works as intended now.
